Documentation/getting-started-guide.md

```diff
@@ -49,6 +62,7 @@ The following commands will create an ACI containing our application and importa
 ```bash
 acbuild begin
 acbuild set-name example.com/hello
+#acbuild annotation add appc.io/executor/supports-systemd-notify true
```

Comment:
This is commented out. As suggested above we could remove this comment if we explain the notification feature in a separate chapter.